config/initializers/cookies_serializer.rb

```diff
@@ -0,0 +1,3 @@
+# Be sure to restart your server when you modify this file.
+
+Rails.application.config.action_dispatch.cookies_serializer = :marshal
```

Comment:
I think the default behavior is marshal and I didn't want to change it to avoid breaking existing sessions.

- Setting it to json will break existing sessions after upgrade
- Using hybrid will break new (json) sessions if the server is rollbacked across this commit.